src/evict/evict_lru.c

```diff
@@ -260,14 +260,17 @@ __wt_evict_server_wake(WT_SESSION_IMPL *session)
     cache = conn->cache;
 
     if (WT_VERBOSE_LEVEL_ISSET(session, WT_VERB_EVICTSERVER, WT_VERBOSE_DEBUG_2)) {
-        uint64_t bytes_inuse, bytes_max;
+        uint64_t bytes_inuse, bytes_max, total_dirty_bytes, total_updates_bytes;
 
         bytes_inuse = __wt_cache_bytes_inuse(cache);
         bytes_max = conn->cache_size;
+        total_dirty_bytes = __wt_cache_dirty_inuse(cache);
+        total_updates_bytes = __wt_cache_bytes_updates(cache);
         __wt_verbose_debug2(session, WT_VERB_EVICTSERVER,
-          "waking, bytes inuse %s max (%" PRIu64 "MB %s %" PRIu64 "MB)",
-          bytes_inuse <= bytes_max ? "<=" : ">", bytes_inuse / WT_MEGABYTE,
-          bytes_inuse <= bytes_max ? "<=" : ">", bytes_max / WT_MEGABYTE);
+          "eviction server waking, bytes max: %.2f(MB) bytes inuse: %.2f(MB) bytes dirty: %.2f(MB) "
+          "bytes updates: %.2f(MB)",
+          (double)bytes_max / WT_MEGABYTE, (double)bytes_inuse / WT_MEGABYTE,
+          (double)total_dirty_bytes / WT_MEGABYTE, (double)total_updates_bytes / WT_MEGABYTE);
     }
```

Comment:
Modified - verbose format, removed precision (I do not see the need to be precise, instead using bytes for dirty and updates seems reasonable)
```
__wt_verbose_debug2(session, WT_VERB_EVICTSERVER,
          "waking, bytes inuse %s max (%" PRIu64 "MB %s %" PRIu64 "MB), bytes dirty %" PRIu64, "(bytes), bytes updates %" PRIu64 "(bytes)",
          bytes_inuse <= bytes_max ? "<=" : ">", bytes_inuse / WT_MEGABYTE,
          bytes_inuse <= bytes_max ? "<=" : ">", bytes_max / WT_MEGABYTE), bytes_dirty, bytes_updates;
```

Reply:
@y123456yz I think you missed this comment.

Reply:
@y123456yz since `bytes inuse` and `bytes_max` should mostly be larger values (as opposed to `bytes dirty` and `bytes updates `. It is better to keep it in MB.